.github/workflows/foreman.yml

```diff
@@ -11,8 +11,24 @@ env:
   BUNDLE_WITHOUT: "console:development:journald"
 
 jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_WITHOUT: assets:console:development:dynflow_sidekiq:ec2:journald:jsonp:libvirt:openid:openstack:ovirt:redis:service:telemetry:vmware
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+          bundler-cache: true
+      - name: Run rubocop
+        run: bundle exec rubocop -P
```

Comment:
When rubocop is >= 1.2 you can use `--format github` to get GH annotations, but we'd need to update theforeman-rubocop from 0.0.6 to 0.1.x. We should add that to the list of updates to perform: https://github.com/theforeman/theforeman-rubocop/pull/8#issuecomment-1825840790

Reply:
Yeah, updating tfm-r would be good.